pyramid_learning_journal/pyramid_learning_journal/views/default.py

```diff
@@ -1,13 +1,14 @@
 """Views for learning journal."""
 from pyramid.response import Response
-from pyramid.view import view_config
 import os
 import io
+import datetime
 
 HERE = os.path.dirname(__file__)
 
 the_date = datetime.datetime.now()
 
+
 def list_view(request):
     """Open home page with list of entries."""
     with io.open(os.path.join(HERE, '../templates/main.html')) as the_file:
```

Comment:
Not something incorrect, but a note to keep in mind. If the only reason you're using that `HERE` variable is to generate a path to point at templates in your templates directory, why not change the variable to just be the path to the templates directory? It looks less confusing that way.